packages/core/utils/src/modules-sdk/load-module-database-config.ts

```diff
@@ -31,7 +31,9 @@ function getDefaultDriverOptions(clientUrl: string) {
   }
 
   if (clientUrl) {
-    return clientUrl.match(/localhost/i) ? localOptions : remoteOptions
+    return clientUrl.match(/localhost|ssl_mode=(disable|false)/i)
```

Comment:
We could check `127.0.0.1` as well. And maybe also if ssl_mode is true?

Reply:
let me improve

Reply:
done 👍